package.json

```diff
@@ -60,15 +68,20 @@
     "react-router-dom": "6.15.0",
     "redux": "4.2.0",
     "regenerator-runtime": "0.13.11",
-    "tinymce": "5.10.7"
+    "tinymce": "5.10.7",
+    "use-deep-compare-effect": "^1.8.1"
```

Comment:
Yep, even with that warning I think that lib worth here. since the `PluggableComponent` props will be an object I tried with [JSON.stringify](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) inside useEffect array dependencies but the component broke sometimes. Another approche was using [`lodash.isEqual`](https://lodash.com/docs/4.17.15#isEqual) to compare props object but did not improve a lot. So at this point this lib is more solid that making something similar here. 

If you think this can be changed please let me know